api/handlers.go

```diff
@@ -0,0 +1,65 @@
+package api
+
+import (
+	"net/http"
+	"strconv"
+	"time"
+
+	"github.com/SkynetLabs/scanner-queue/database"
+	accdb "github.com/SkynetLabs/skynet-accounts/database"
+	"github.com/julienschmidt/httprouter"
+	skyapi "gitlab.com/SkynetLabs/skyd/node/api"
+)
+
+func (api *API) ScanPOST(w http.ResponseWriter, r *http.Request, ps httprouter.Params) {
+	skylink := ps.ByName("skylink")
+	if skylink == "" {
+		skyapi.WriteError(w, skyapi.Error{"missing required parameter 'skylink'"}, http.StatusBadRequest)
+		return
+	}
+	if !accdb.ValidSkylinkHash(skylink) {
+		skyapi.WriteError(w, skyapi.Error{"invalid skylink"}, http.StatusBadRequest)
+		return
+	}
+	sl := database.Skylink{
+		Skylink:   skylink,
+		Timestamp: time.Now().UTC(),
+		Scanned:   false,
```

Comment:
Based on our discussion earlier, should we make this `scannedStatus: string`? So it could be `initialized, scanning, complete`?

That or add another boolean field `inProgress`.

Reply:
I prefer the `new`/`scanning`/`complete` approach.